docs/homeassistant/homeassistant-official.md

```diff
@@ -2,18 +2,34 @@
 
 To integrate your Z-Wave components you can use either the official Home Assistant [Z-Wave JS integration](https://www.home-assistant.io/integrations/zwave_js) and/or MQTT discovery.
```

Comment:
```suggestion
To integrate your Z-Wave components you can use either the official Home Assistant [Z-Wave integration](https://www.home-assistant.io/integrations/zwave_js) or MQTT discovery.
```

1. It's now called the Z-Wave integration. The [Usage docs](https://github.com/zwave-js/zwave-js-ui/blob/30d020827084867fcc10746445862b8eacff3375/docs/usage/setup.md?plain=1#L231) were already updated to reflect this.
2. It should not be suggested to run the integration and MQTT discovery at the same time since that results in duplicate devices.

Reply:
Should probably not have included the "and" to begin with. Good catch.